### PR TITLE
TDRD-148: batch the files status updates (batches of 10,000)

### PR DESCRIPTION
**Pull request by MarkKingsbury**
for 10,000 files testing the step to update file statuses includes 70,000 in one call, generating lots of waits for write ahead log activity and timeouts... takes around 8 mins.

batching into groups of 10,000 in each call runs without noticeable write ahead log waits and each call completes in around 30 secs.

### CODE REVIEW

File: src/main/scala/uk/gov/nationalarchives/api/update/Lambda.scala [hunk 1]
```diff
@@ -125,6 +129,6 @@ class Lambda {
     } yield {
       output.write(inputStream.readAllBytes())
     }
-    Await.result(result, 90.seconds)
+    Await.result(result, 600.seconds)
```
Comment (TomJKing) [inline src/main/scala/uk/gov/nationalarchives/api/update/Lambda.scala:132]: With the batching, does this await still need to be this long?

**Review by TomJKing — COMMENTED** [review]
Looks good. Thanks. Just one minor question.

File: src/main/scala/uk/gov/nationalarchives/api/update/Lambda.scala (hunk 1 above)
Comment (MarkKingsbury) [reply src/main/scala/uk/gov/nationalarchives/api/update/Lambda.scala:132]: It's taking just over 5 mins so I guess depends how much spare space we want - could drop it to 7 or 8 mins?

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks.

When its in intg, should probably just do a manual test to make sure that everything still works OK.

**Pull request merged**